### ISSUE TITLE
Test error after upgrading to 0.3.2 : `(ArgumentError) no push or navigation command found within JS commands`

### ISSUE THREAD

**Issue by joshk** [germsvel/phoenix_test#122]
After upgrade `phoenix_test` to 0.3.2 I have two tests failing for the same reason.

```
1) test new create with JITP (NervesHubWeb.Live.Org.CertificateAuthoritiesTest)
     test/nerves_hub_web/live/org/certificate_authorities_test.exs:120
     ** (ArgumentError) no push or navigation command found within JS commands: [["dispatch",{"event":"ca:new:jitp"}]]
     code: |> check("Enable Just In Time Provisioning")
     stacktrace:
       (phoenix_live_view 0.20.17) lib/phoenix_live_view/test/live_view_test.ex:1102: Phoenix.LiveViewTest.call/2
       (phoenix_test 0.3.2) lib/phoenix_test/live.ex:138: PhoenixTest.Live.check/2
       test/nerves_hub_web/live/org/certificate_authorities_test.exs:146: (test)
```

This is the full test here: https://github.com/nerves-hub/nerves_hub_web/blob/main/test/nerves_hub_web/live/org/certificate_authorities_test.exs#L119-L166

And here is the full html: https://github.com/nerves-hub/nerves_hub_web/blob/main/lib/nerves_hub_web/live/org/certificate_authority_templates/new_ca_template.html.heex (the checkbox in question: https://github.com/nerves-hub/nerves_hub_web/blob/main/lib/nerves_hub_web/live/org/certificate_authority_templates/new_ca_template.html.heex#L73)

Am I doing something wrong?

**Comment by germsvel** [comment]
@joshk thanks for opening the issue. I'm looking at the code you shared, and I can't really see how things would've changed in 0.3.2. I'll try to take a deeper look at the changes we made to see if anything comes up. 

In the meantime, is there a way you could try to reproduce the error in a simpler phoenix app? something I can pull down and play around with? 

Another question: do you know if you also updated LiveView when you upgraded phoenix test 0.3.2? The error you're seeing looks like something that would come out of LiveView or LiveViewTest, which makes me think something else might've changed.

**Comment by germsvel** [comment]
@joshk I'm going to close this for now since it seems like there's no way for me to reproduce this. 

If you find the time to create a simple Phoenix app that reproduces the error, I'd be happy to reopen this!

**Comment by joshk** [comment]
Totally understand. We are stuck on 0.3.1 for now, until I can show the issue in a small app.

**Comment by joshk** [comment]
> Another question: do you know if you also updated LiveView when you upgraded phoenix test 0.3.2? The error you're seeing looks like something that would come out of LiveView or LiveViewTest, which makes me think something else might've changed.

I forgot to reply and say that no, we haven't changed LiveView versions, this is isolated to only PhoenixTest.

**Comment by germsvel** [comment]
I'll reopen this in case you get time to create a simple app to try to reproduce what's going on there.

**Comment by germsvel** [comment]
@joshk while solving [another issue](https://github.com/germsvel/phoenix_test/issues/133#issuecomment-2404566999), I figured out why your test started breaking after 0.3.2

In 0.3.2, we introduced the ability for checkboxes to handle `phx-click` attributes. The intention was for them to be handled outside of forms, but technically, we just handle the `phx-click` first. 

LiveViewTest (what PhoenixTest is using underneath) doesn't support testing `phx-click`s that only have `JS.dispatch/1` in them. So, the combination of PhoenixTest now attempting to handle your `phx-click` and of LiveViewTest not handling `JS.dispatch/1` leads to your test now giving you that "no push or navigation command found within JS commands"

I think PhoenixTest should probably ignore any JS commands that don't have a push or navigate in them. I'll see how to work on this.
